cohort_schedules/C07_classroom.md

```diff
@@ -1,39 +1,126 @@
+
+Major topics of study include:
+
+| Writing Software        | Being a Developer
+|-------------------------|-------------------
+| Ruby                    | Leadership & Inclusion
+| HTML & CSS              | Test-Driven Development
+| Ruby on Rails           | Pair Programming
+| Web APIs                | Agile Practices
+| JavaScript              | Networking
+| CS Fundamentals         | Career Readiness
+
+## Expected Outcomes
+
+### Ruby
+
+Ruby is a dynamically typed, interpreted, object-oriented language. In addition to serving as a great introduction to programming, Ruby is the foundation of the Ruby on Rails framework, and is used for most of our CS Fundamentals curriculum.
+
+Students should be able to:
+
+* Write complex command line programs in Ruby
+* Use object-oriented programming principals to build scalable, maintainable programs
+* Process and build relations between data from files
+* Verify code from the outset using test-driven development
+
+### HTML & CSS
+
+HTML and CSS are two of the core technologies on which the web is built, allowing web developers to structure content in a hierarchy and specify how it should be presented to the user. Both are foundational for much of the rest of the course.
+
+Students should be able to:
+
+* Read and write HTML and CSS
+* Utilize semantic tags to provide clear structure to HTML
+* Manage the layout of a complex web page using CSS
+
+### Ruby on Rails
+
+Ruby on Rails is a robust and comprehensive MVC framework for building web servers. We use Rails to address a wide range of topics, from HTTP, REST and web standards to database operations. This unit also includes a large team project completed over two weeks.
+
+Students should be able to:
+
+* Build and deploy a complete website, from the HTML & CSS on the page to the backing database
+* Model complex real-world relations using ActiveRecord
+* Understand the basics of web security and user authentication
+* Use Git, TDD and agile practices to coordinate a large team
+
+### Web APIs
+
+APIs allow programs to talk to each other, and have become an important part of the modern web. We focus both on consuming third party APIs and on producing our own APIs, laying the foundation for future front-end scripting work.
+
+Students should be able to:
+
+* Explore and utilize third party APIs
+* Build an API to expose their own data
+
+### JavaScript
+
+As the language of the web, a strong understanding of JavaScript is essential for any prospective web developer. In addition to core language features, we'll cover some of the excellent libraries that enable modern web apps such as jQuery, Underscore and the Backbone framework.
```

Comment:
I might remove "strong" because we don't spend a ton of time here so we don't necessary set them up for what I'd refer to as a "strong" understanding.